src/trace_processor/metrics/sql/android/android_blocking_calls_cuj_per_frame_metric.sql

```diff
@@ -87,7 +87,7 @@ WHERE
       OR
       -- frame end within cuj
       (frame.ts_end >= cuj.ts AND frame.ts_end <= cuj.ts_end)
-   );
+   ) GROUP BY frame_id;
```

Comment:
If you're grouping by frame_id, please be explicit in the aggregate functions you are taking with min/max/other aggregate functions. Implicitly relying on sqlite to pick which row is chosen is not a good idea.

Reply:
Please leave a comment then.